packages/server/graphql/types/PokerMeeting.ts

```diff
@@ -39,6 +43,31 @@ const PokerMeeting = new GraphQLObjectType<any, GQLContext>({
           .load({teamId, meetingType: MeetingTypeEnum.poker})
       }
     },
+    story: {
+      type: Story,
+      description: 'A single story created in a Sprint Poker meeting',
+      args: {
+        storyId: {
+          type: GraphQLNonNull(GraphQLID)
+        }
+      },
+      resolve: async ({teamId}, {storyId}, {authToken, dataLoader}) => {
+        const isJiraId = await isValidJiraId(storyId, teamId, {authToken, dataLoader})
+        if (isJiraId) {
+          const [cloudId, key] = getJiraCloudIdAndKey(storyId)
+          return {
```

Comment:
remember, whatever the return value is must fulfill the `Story` contract.
For example, here we're returning a JiraIssue, but a JiraIssue has a nonnull field called cloudName.
that means the following would throw an error
```gql
story(storyId) {
  id
  cloudName
}
```

try to fulfill the Story contract by resolving all required fields.
see https://github.com/ParabolInc/parabol/blob/de8d951f698f01dd1a1decfe0e3e1b605d4520ae/packages/server/graphql/types/EstimateStage.ts#L91 for a hint